README.md

```diff
@@ -79,13 +79,17 @@ These can be installed with [homebrew](https://brew.sh/) on MacOS
 
 ```bash
 brew install node
-brew install nvm
+brew install nvm # Make sure to add nvm to your path
 brew install jq
+nvm install 20 # .github/workflows/pr.yml users node version 20
```

Comment:
on line 100 we run nvm use --lts, can you update that, too?

Reply:
🤔 so on both this line and line 100 should it be `nvm use 20` or `nvm use --lts`? I wasn't familiar with the `--lts` option

Reply:
I think nvm use 20 is probably preferable? The latest stable release is probably newer but we could guarantee that node version 20 will work as expected and has been tested

Reply:
@anayeaye done, should I go ahead and merge?

Reply:
thanks! I'll merge in just to save you another trip :)